### DISCUSSION TITLE
How to exclude a dependency for a feature in manifest mode?

### DISCUSSION THREAD

**Discussion by aungthetnaing** [microsoft/vcpkg#40436] [Q&A]
Is there any way to remove a dependency if a feature is defined?

For example, I have libA and libB.

In default, I would like to use libA.

But, if a feature is defined, I would like to use libB and libA should be removed. libA and libB conflicts each other.

**Comment by dg0yt** [comment]
> Is there any way to remove a dependency if a feature is defined?

No.

> libA and libB conflicts each other.

vcpkg can't resolve conflicts aka alternatives.

**Comment by autoantwort** [comment]
As a workaround you could have a feature for libA and one for libB and error out if none or both are defined.
